### PR TITLE
Add project

### PR DESCRIPTION
**Pull request by sevinchek**
## My first Project as a Code Reviewer

In this branch I:
- Copy project from Admissions Trial
- Set up a Linter
- Added a descriptive README
- Added an App Screenshot

### CODE REVIEW

**Comment by sevinchek** [comment]
Hi @sevinchek,

Good Job so far 👍, but you need to check the Linter errors.
I will describe the errors in the next comments.👇

**Comment by sevinchek** [comment]
You should check the errors from Stylelint and correct them. I suggest you to install Stylelint in your code editor so you can correct the Style errors in your code editor and cannot get errors when you pull request the next time. 🙌

**Comment by sevinchek** [comment]
There is an open-element (ul) as Webhint Linter shows in the Details 🤔. It would be better to check it and close it 🤓. Please check the other errors and correct them ✅.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+/*
+@media screen and (min-width: 1281px) {
+  .sidenav {padding-top: 15px;}
+  .sidenav a {font-size: 18px;}
+  body {
+    background-color: white;
+  }
+}*/
```
Comment (bolah2009) [inline css/style.css:71]: This is an example inline comment. 🙂 

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉 

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi @sevinchek,

Good job so far!
You did a good job explaining the issue and addressing the comments politely and clear. 👏🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️


1. Let's use the GitHub inline comment. See the example inline comment review below.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comment under one single summary like this current review. 😉 


Cheers and Happy coding! 👏🏽 👏🏽 👏🏽 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (sevinchek) [inline index.html:30]: You should delete commented code to have a clean code ✅.

**Review by sevinchek — COMMENTED** [review]
Hi @sevinchek,

Please check my suggestion below. 👇👇

Feel free to leave any questions or comments in the PR thread if something is not 100% clear ✌️.

**Pull request merged**